packages/engine/src/avatar/functions/avatarFunctions.ts

```diff
@@ -185,15 +194,22 @@ export const animateModel = (entity: Entity) => {
     .play()
 }
 
-export const setupAvatarMaterials = (root) => {
+export const setupAvatarMaterials = (entity, root) => {
   const materialList: Array<MaterialMap> = []
 
   setObjectLayers(root, ObjectLayers.Avatar)
   root.traverse((object) => {
     if (object.isBone) object.visible = false
     if (object.material && object.material.clone) {
       const material = object.material.clone()
-      setupHeadDecap(root, material)
+
+      if (hasComponent(entity, NameComponent)) {
```

Comment:
we can more easily check this by checking `entity === Engine.instance.currentWorld.localClientEntity`

Reply:
> we can more easily check this by checking `entity === Engine.instance.currentWorld.localClientEntity`

Awesome! Thanks for the letting me know!